src/db.ts

```diff
@@ -27,6 +31,7 @@ export interface IConfig {
 }
 
 export interface IEvent {
+  outbox: any
```

Comment:
I think this should be

```suggestion
  outbox?: boolean
```

or does typescript support

```suggestion
  outbox?: true
```

as it's really only defined as `true` or not defined at all.

Reply:
Thank you for providing context as to what the type should be, I wasn't able to figure it out for the life of me.

I will type it as `outbox?: true`